cg/meta/upload/scout/mip_config_builder.py

```diff
@@ -131,6 +132,16 @@ def include_sample_files(self, config_sample: ScoutMipIndividual) -> None:
                 hk_tags=self.sample_tags.chromograph_sites, sample_id=sample_id
             )
         )
+        config_sample.reviewer.alignment = self.fetch_sample_file(
+            hk_tags=self.sample_tags.reviewer_alignment, sample_id=sample_id
+        )
+        config_sample.reviewer.alignment_index = self.fetch_sample_file(
+            hk_tags=self.sample_tags.reviewer_alignment_index, sample_id=sample_id
+        )
+        config_sample.reviewer.vcf = self.fetch_sample_file(
+            hk_tags=self.sample_tags.reviewer_vcf, sample_id=sample_id
+        )
+        config_sample.reviewer.catalog = self.fetch_file_from_hk(hk_tags=self.case_tags.str_catalog)
```

Comment:
Does Scout want this per sample or is it per case? Should be the same catalogue for an analysis.

Reply:
Scout wants it per sample but we only store it once in MIP ("case level"). So this is strying to move that file in under sample in the scout load config.